sqldb/sqlc/migrations/000003_invoice_events.up.sql

```diff
@@ -0,0 +1,60 @@
+-- invoice_event_types stores the different types of events that can be emitted 
+-- for invoices.
+CREATE TABLE IF NOT EXISTS invoice_event_types(
+    id INTEGER PRIMARY KEY,
+
+    description TEXT NOT NULL
```

Comment:
The idea is for external tools to be able to get their info from here instead of having to keep their apps in sync with the latest LND Go code for this.

Reply:
I think it just makes it easier to standardize the descriptions vs defining them at query time. The id sort of works as the enum.

Reply:
In the future if we want very strict typing here, we can make a set "enum values" table, then point to that itself (all items unique, essentially pre-populated). 

You can also do stuff like:
```sql
type CHECK(type in ('THIS', THAT'))
```
etc